projects/aca-content/src/lib/components/knowledge-retrieval/search-ai/search-ai-results/search-ai-results.component.spec.ts

```diff
@@ -53,20 +58,24 @@ describe('SearchAiResultsComponent', () => {
     });
 
     fixture = TestBed.createComponent(SearchAiResultsComponent);
+    fixture.detectChanges();
     component = fixture.componentInstance;
-    spyPerformAiSearch = spyOn(component, 'performAiSearch');
     component.ngOnInit();
   });
 
   afterEach(() => {
     mockQueryParams = new Subject<Params>();
-    knowledgeRetrievalNodes = '{"isEmpty":"false","nodes":[{"entry":{"id": "someId","isFolder":"true"}}]}';
+    knowledgeRetrievalNodes = knowledgeRetrievalNode;
 
     fixture.destroy();
     component.ngOnDestroy();
   });
 
-  describe('query params change', () => {
+  describe('skeleton loader', () => {
+    beforeEach(() => {
+      spyPerformAiSearch = spyOn(component, 'performAiSearch');
```

Comment:
Here similar comment like for searchAiService variable - you assign spyPerformAiSearch here but it's declared in top describe so could you please move declaration of this variable also to this inner describe as it's used only here?